.changeset/slow-items-heal.md

```diff
@@ -0,0 +1,34 @@
+---
+"astro": minor
```

Comment:
Not sure if this really needs to be a minor or if we could get it out sooner, cause it is behind an experimental flag. I would prefer the second one.